osi_hostvehicledata.proto

```diff
@@ -337,6 +342,31 @@ message HostVehicleData
         optional GeodeticPosition geodetic_position = 3;
     }
 
+    //
+    //\brief Current calculated and estimated acceleration, velocity and orientation rate on Inertial Measurement Unit measurements and related sensors.
+    //
+    //This message contains the most accurate information the vehicle knows about its velocity, 
```

Comment:
// This
Space is missing

Reply:
Thank you for the review. 
Spaces have been included.